ui/src/app/pages/login/account-creation/account-creation.spec.tsx

```diff
@@ -2,23 +2,27 @@ import {mount} from 'enzyme';
 import * as fp from 'lodash/fp';
 import * as React from 'react';
 
-import {AccountCreation, AccountCreationProps, AccountCreationState} from './account-creation';
-import {AccountCreationOptions} from './account-creation-options';
 import {serverConfigStore} from 'app/utils/navigation';
-import {getEmptyProfile} from 'app/pages/login/test-utils';
 import {Profile} from 'generated/fetch';
+import {createEmptyProfile} from 'app/pages/login/sign-in';
+import {AccountCreation, AccountCreationProps} from './account-creation';
+import {AccountCreationOptions} from './account-creation-options';
 
 let props: AccountCreationProps;
 const component = () => {
   return mount(<AccountCreation {...props}/>);
 };
 
-const defaultConfig = {gsuiteDomain: 'researchallofus.org', enableNewAccountCreation: false};
+const defaultConfig = {
+  gsuiteDomain: 'researchallofus.org',
+  enableNewAccountCreation: false,
+  requireInstitutionalVerification: false
+};
 
 beforeEach(() => {
   serverConfigStore.next(defaultConfig);
   props = {
-    profile: getEmptyProfile(),
```

Comment:
`getEmptyProfile()` was a test-only copy of `createEmptyProfile()` which I removed